packages/strapi-design-system/src/TextInput/TextInput.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, FieldLabel, FieldHint, FieldError, FieldInput, FieldAction } from '../Field';
+import { Stack } from '../Stack';
+import { Row } from '../Row';
+import { Box } from '../Box';
+import styled from 'styled-components';
+
+const TextareaWrapper = styled.div`
+  & textarea {
+    height: ${80 / 16}rem;
+  }
+`;
+
+export const TextInput = React.forwardRef(
+  ({ leftAction, rightAction, name, hint, error, label, action, ...props }, ref) => {
+    return (
+      <TextareaWrapper>
+        <Field name={name} hint={hint} error={error}>
+          <Stack size={1}>
+            <Row cols="auto auto 1fr" gap={1}>
+              <FieldLabel>{label}</FieldLabel>
+              {action && <Box paddingLeft={1}>{action}</Box>}
```

Comment:
The difference between `leftAction, rightAction and action` props is not very clear for me to be honest. In which use cases should we use `leftAction/rightAction` vs `action`.

Reply:
Oh yeah I understand 😊 . Left/ Right actions are for the input component (like the "eye" for the password, or the "magnifier" for the search input"). The action is for the tooltip information.

Do you have something in mind to adjust this?

Reply:
WDYT about `labelAction`

Reply:
Let's try with that 👍🏻